scripts/globals/spells/enfeebling_spell.lua

```diff
@@ -131,7 +131,7 @@ xi.spells.enfeebling.calculatePotency = function(caster, target, spellId, spellE
             statDiff = caster:getStat(statUsed) - target:getStat(xi.mod.MND)
 
             if spellId == xi.magic.spell.BLIND_II then
-                potency = utils.clamp(statDiff * 0.375 + 49, 15, 90) -- Values from JP wiki: http://wiki.ffo.jp/html/3449.html
+                potency = utils.clamp(statDiff * 0.375 + 49, 19, 94) -- Values from JP wiki: http://wiki.ffo.jp/html/3449.html
```

Comment:
If I'm understanding the math right over on the multiple wiki's, the reason the rang is 19-94 is simply because that's the most the dStat can push (-80 to +120). I don't see why we'd need to clamp the range at this spot if our other steps are correct. /shrug

but since the clampage already existed, I'll stamp the range change.